Tests/SentryTests/SentryCrash/CrashReport.swift

```diff
@@ -9,7 +9,7 @@ extension XCTestCase {
     
     func givenStoredSentryCrashReport(resource: String) throws {
         let jsonData = try jsonDataOfResource(resource: resource)
-        jsonData.withUnsafeBytes { ( bytes: UnsafeRawBufferPointer) -> Void in
+        jsonData.withUnsafeBytes { ( bytes: UnsafeRawBufferPointer) in
```

Comment:
The lint bot just changed this, duno why.

Reply:
Swift block dont have the return type, all good.